src/client/dialog-demo-mui/styles.css

```diff
@@ -0,0 +1,34 @@
+/* needed to make consistent test snapshots across OSs */
+body {
+  font-family: Arial !important;
+}
+
+/*
+CSSTransitionGroup styling
```

Comment:
we probably don't need these since you didn't use react-transition-group in the MUI example